modules/ui/src/app/app.component.html

```diff
@@ -189,6 +189,28 @@ <h1 class="main-heading">Testrun</h1>
             >start testing</a
           >.
         </app-callout>
+        <app-callout
+          role="alert"
+          aria-live="assertive"
+          [type]="CalloutType.Check"
+          *ngIf="
+            vm.hasConnectionSettings === true &&
+            vm.hasDevices === true &&
+            vm.hasRiskProfiles === false &&
+            isTestrunInProgress(vm.systemStatus)
```

Comment:
In this case, status should be in Monitoring or In Progress

Reply:
Clarified and updated to show only In Progress status